README.md

```diff
@@ -14,6 +14,12 @@ The [contributing documentation](CONTRIBUTING.md) covers licencing and the usual
 
 **Do Not Ever Update Quay Labels Based On Your Local Development Branch. Only from master!**
 
+### Release names
+
+When [adding releases to channels](#add-releases-to-channels) or [blocking edges](#block-edges), releases are given by name.
+Architecture-agnostic names like 4.2.14 will apply to all images with that exact name in the `version` property of the `release-metadata` file included in the release image.
+Names with [SemVer build metadata][semver-build] like 4.2.14+amd64 will apply only to releases whose exact name in the `version` property matches the release with the build metadata removed (e.g. 4.2.14) but whose referenced [image architecture][image-arch] matches the given build metadata.
```

Comment:
The current arch specific releases have `4.2.12-s390x` as name, so we should use this an example instead of `4.2.14+amd64`?

Reply:
Also `Names with [SemVer build metadata][semver-build] like 4.2.14+amd64 will apply only to releases whose exact name in the `version` property matches the release with the build metadata removed (e.g. 4.2.14) but whose referenced [image architecture][image-arch] matches the given build metadata.` is a long sentence and we should break this down.

Reply:
> The current arch specific releases have 4.2.12-s390x as name, so we should use this an example instead of 4.2.14+amd64?

`-` (prerelease name) instead of `+` (build metadata) was an artifact of old ART naming.  This commit has nothing to do with prerelease names, and folks handling those will still need to use the full name which was baked into the release (e.g. [this][1] is still correct).  With this PR, you could use `4.2.11-s390x+390x` as well, but since there are no releases named `4.2.11-s390x` on other arches, you don't have to.

[1]: https://github.com/openshift/cincinnati-graph-data/blob/8e681e27f8f725b8506839a39803ebc0e5aaf575/channels/stable-4.2.yaml#L20

Reply:
Suggestions around breaking the sentences: 
* When adding releases to channels or blocking edges, releases should be given by name. 
* Architecture-agnostic names like 4.2.14 will apply to all images with that exact name in the version property of the release-metadata file included in the release image. 
* Names with SemVer build metadata like 4.2.14+amd64 will match to releases stings without the build metadata (e.g. 4.2.14)  in version property along with the referenced image architecture matches the given build metadata.

Reply:
>...is a long sentence and we should break this down.

I've pushed beb55be -> 3c49e24, pulling the 4.2.14 examples down into subsequent sentences.  How does that look?